db/schema.rb

```diff
@@ -10,14 +10,45 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_01_13_005107) do
+ActiveRecord::Schema.define(version: 2018_11_09_013523) do
```

Comment:
It looks like when you did `git checkout`, master wasn't up-to-date. If you can make sure that's up to date, then do the same `git checkout master db/schema.rb`, and then make sure the version matches `2019_01_13_005107`?